### ISSUE TITLE
Issue with linter and mat table

### ISSUE THREAD

**Issue by zominym** [yeonjuan/html-eslint#102]
Hi, thanks for the plugin, it's great.

However, there seems to be a mishap with the handling of [material tables](https://material.angular.io/components/table/examples#table-basic),
this snippet will not pass the indent and closing-tag rules when it should :

``` html
<table>
    <ng-container>
        <th>world</th>
        <td>hello</td>
    </ng-container>
</table>
```

Is there something I'm missing or is that an issue with the plugin?

**Comment by yeonjuan** [comment]
Hi @zominym 
There is some issue with the parser, it too strictly follows HTML spec so it makes some problems.
I'll work on th fix some kinds of problems related to the parser.
